api/restapi/configure_empty.go

```diff
@@ -115,10 +115,10 @@ func setupMiddlewares(handler http.Handler) http.Handler {
 func setupGlobalMiddleware(handler http.Handler) http.Handler {
 	return http.HandlerFunc(func(w http.ResponseWriter, r *http.Request) {
 		// Shortcut helpers for swagger-ui
-		if r.URL.Path == "/swagger-ui" {
-			http.Redirect(w, r, "/swagger-ui/", http.StatusFound)
-			return
-		}
+		// if r.URL.Path == "/swagger-ui" {
```

Comment:
Pls remove the uncommented code.

Reply:
Can we please also remove the redirect here?
https://github.com/keptn/keptn/blob/master/configuration-service/restapi/configure_configuration_service.go
https://github.com/keptn/keptn/blob/master/mongodb-datastore/restapi/configure_mongodb_datastore.go